### ISSUE TITLE
[auto-generated:25f7087] Unspecified error

### ISSUE THREAD

**Issue by jb-error-reporter** [JetBrains-Research/IntelliJDeodorant#76]
- Plugin Name: IntelliJDeodorant
- Plugin Version: 2019.3-1.2
- OS Name: Linux
- Java Version: 11.0.6
- Java VM Vendor: JetBrains s.r.o
- App Name: IDEA
- App Full Name: IntelliJ IDEA
- App Version Name: IntelliJ IDEA
- Is EAP: false
- App Build: IU-193.6911.18
- App Version: 2019.3.4
- Last Action: ActivateIntelliJDeodorantToolWindow
- User's Permanent Installation ID: be356905-2006-424a-87c3-391608f11635

```
java.lang.NullPointerException
	at org.jetbrains.research.intellijdeodorant.core.ast.decomposition.cfg.PDG.antiDependenceSearch(PDG.java:652)
	at org.jetbrains.research.intellijdeodorant.core.ast.decomposition.cfg.PDG.createDataDependencies(PDG.java:555)
	at org.jetbrains.research.intellijdeodorant.core.ast.decomposition.cfg.PDG.<init>(PDG.java:41)
	at org.jetbrains.research.intellijdeodorant.JDeodorantFacade.processMethod(JDeodorantFacade.java:94)
	at org.jetbrains.research.intellijdeodorant.JDeodorantFacade.getExtractMethodRefactoringOpportunities(JDeodorantFacade.java:83)
	at org.jetbrains.research.intellijdeodorant.ide.ui.ExtractMethodPanel$1.lambda$run$4(ExtractMethodPanel.java:207)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:796)
	at org.jetbrains.research.intellijdeodorant.ide.ui.ExtractMethodPanel$1.run(ExtractMethodPanel.java:206)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:888)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:163)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:585)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:531)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:150)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.lambda$run$0(CoreProgressManager.java:402)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:221)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:402)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by jb-error-reporter** [comment]
- Plugin Name: IntelliJDeodorant
- Plugin Version: 2019.3-1.2
- OS Name: Mac OS X
- Java Version: 11.0.6
- Java VM Vendor: JetBrains s.r.o
- App Name: IDEA
- App Full Name: IntelliJ IDEA
- App Version Name: IntelliJ IDEA
- Is EAP: false
- App Build: IC-193.6911.18
- App Version: 2019.3.4
- Last Action: 
- User's Permanent Installation ID: c14894fe-42e0-4544-856f-f295202a9293

```
java.lang.NullPointerException
	at org.jetbrains.research.intellijdeodorant.core.ast.decomposition.cfg.PDG.antiDependenceSearch(PDG.java:652)
	at org.jetbrains.research.intellijdeodorant.core.ast.decomposition.cfg.PDG.createDataDependencies(PDG.java:555)
	at org.jetbrains.research.intellijdeodorant.core.ast.decomposition.cfg.PDG.<init>(PDG.java:41)
	at org.jetbrains.research.intellijdeodorant.JDeodorantFacade.processMethod(JDeodorantFacade.java:94)
	at org.jetbrains.research.intellijdeodorant.JDeodorantFacade.getExtractMethodRefactoringOpportunities(JDeodorantFacade.java:83)
	at org.jetbrains.research.intellijdeodorant.ide.ui.ExtractMethodPanel$1.lambda$run$4(ExtractMethodPanel.java:207)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:796)
	at org.jetbrains.research.intellijdeodorant.ide.ui.ExtractMethodPanel$1.run(ExtractMethodPanel.java:206)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:888)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:163)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:585)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:531)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:150)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.lambda$run$0(CoreProgressManager.java:402)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:221)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:402)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by jb-error-reporter** [comment]
- Plugin Name: IntelliJDeodorant
- Plugin Version: 2019.3-1.2
- OS Name: Mac OS X
- Java Version: 11.0.5
- Java VM Vendor: JetBrains s.r.o
- App Name: IDEA
- App Full Name: IntelliJ IDEA
- App Version Name: IntelliJ IDEA
- Is EAP: false
- App Build: IU-193.6494.35
- App Version: 2019.3.3
- Last Action: null
- User's Permanent Installation ID: 980d8dda-857b-4e1d-8f4b-bc747f33df53

```
java.lang.NullPointerException
	at org.jetbrains.research.intellijdeodorant.core.ast.decomposition.cfg.PDG.antiDependenceSearch(PDG.java:652)
	at org.jetbrains.research.intellijdeodorant.core.ast.decomposition.cfg.PDG.createDataDependencies(PDG.java:555)
	at org.jetbrains.research.intellijdeodorant.core.ast.decomposition.cfg.PDG.<init>(PDG.java:41)
	at org.jetbrains.research.intellijdeodorant.JDeodorantFacade.processMethod(JDeodorantFacade.java:94)
	at org.jetbrains.research.intellijdeodorant.JDeodorantFacade.getExtractMethodRefactoringOpportunities(JDeodorantFacade.java:83)
	at org.jetbrains.research.intellijdeodorant.ide.ui.ExtractMethodPanel$1.lambda$run$4(ExtractMethodPanel.java:207)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:796)
	at org.jetbrains.research.intellijdeodorant.ide.ui.ExtractMethodPanel$1.run(ExtractMethodPanel.java:206)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:888)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:163)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:585)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:531)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:150)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.lambda$run$0(CoreProgressManager.java:402)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:221)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:402)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:238)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by onewhl** [comment]
The issue was fixed in 2019.3-1.3, please update the plugin.
